src/components/RichTextConverters/upload.ts

```diff
@@ -33,24 +33,10 @@ const upload = ({ node }: { node: SerializedUploadNode }): string => {
   }
 
   return container(`
-    <div class="usa-alert usa-alert--info usa-alert--slim usa-alert--no-icon">
-    <div class="usa-alert__body">
-      <h4 class="usa-alert__heading display-flex flex-row flex-align-center">
-        ${fileIcon} Download File
-      </h4>
-      <p class="usa-alert__text">
-        <a class="usa-link" href="${url}" download="true">
-          ${value.altText || value.filename}
-        </a>
-      </p>
-      <div class="usa-alert__text display-flex flex-column flex-align-end maxw-full">
-        <p class="usa-alert__text">
-          ${formatMimeType(value.mimeType)}
-          ${formatBytes(value.filesize)}
-        </p>
-      </div>
+    <div>
+      <p><a class="usa-link" href="${url}" download="true">${value.altText || value.filename} [${formatMimeType(value.mimeType)} - ${formatBytes(value.filesize)}]</a>
```

Comment:
Add the link icon to the component 

<img width="514" height="75" alt="Image" src="https://github.com/user-attachments/assets/c7a4cba3-d5e4-40b2-9256-144059926cb4" />